google_compute_engine_oslogin/pam_module/pam_oslogin_login.cc

```diff
@@ -102,10 +108,138 @@ PAM_EXTERN int pam_sm_acct_mgmt(pam_handle_t *pamh, int flags, int argc,
       remove(users_filename.c_str());
     }
     pam_syslog(pamh, LOG_INFO,
-               "Denying login permission for organization user %s.", user_name);
+               "Denying login permission for organization user %s.",
+               user_name);
 
     pam_result = PAM_PERM_DENIED;
   }
   return pam_result;
 }
+PAM_EXTERN int pam_sm_setcred(pam_handle_t * pamh, int flags, int argc,
+                              const char **argv)
+{
+            return PAM_SUCCESS;
+}
+
+
+PAM_EXTERN int pam_sm_authenticate(pam_handle_t * pamh, int flags,
+                                                   int argc, const char **argv)
```

Comment:
This also has incorrect indentation. In general, comments apply to all of the changes.